test/verify/machineslib.py

```diff
@@ -1134,7 +1134,7 @@ def installWithErrorTest(dialog):
         # This is applicable only for the following test so let's keep it last,
         # in order to allow the rest of the tests to run faster with QEMU KVM
         # Run modprobe -r in retry loop because it fails sometimes with 'Module kvm is in use'
-        wait(lambda: "Module kvm is in use" not in self.machine.execute("modprobe -r kvm_intel && modprobe -r kvm"))
+        wait(lambda: "Module kvm is in use" not in self.machine.execute("(modprobe -r kvm_intel && modprobe -r kvm) || virsh list --all"))
```

Comment:
This change needs to go into the first commit, it's out of place for this one. But more importantly, this acts like an `|| t rue`, so it means the loop will end as soon as e. g. removing kvm_intel fails once (as that's not caught by the message check). This might explain why you didn't get a timeout for this any more. How about something like this:
```py
try:
    wait(lambda: self.machine.execute("modprobe -r kvm_intel && modprobe -r kvm"))
except Error:
    print(self.machine.execute("virsh list --all"))
    raise
```
This should provide the debug output and still be more robust.

Reply:
virsh is not good enough source I think. I can be VM on system or session connection. Also something can be blocking the kvm module without libvirt being aware of it.
Anyway, virsh list returns not VM just for the record.
Removed the loop and added a lsmod in the catch, I think it might give some info regarding what is using the kvm module if it fails again.

Reply:
Sure, let's start with that. Just about the only thing that could use kvm should be exposed with `pgrep -af qemu || true`, so if we run into troubles, we can extend it later. Thanks!

Reply:
So changed the except print statement to contain the ouptput of the process grep for qemu and reposted. This should not robustify it of course but probably it will help with debugging.